include/rtc/configuration.hpp

```diff
@@ -87,6 +87,10 @@ struct RTC_CPP_EXPORT Configuration {
 
 	// Local maximum message size for Data Channels
 	optional<size_t> maxMessageSize;
+
+	// Certificates and private keys
+	optional<string> certPem;
+	optional<string> keyPem;
```

Comment:
You should make the configuration fields consistent with existing ones in `WebSocketConfiguration` and `WebSocketServerConfiguration`. The fields are called `certificatePemFile`, `keyPemFile`, and `keyPemPass`, they allow the user to pass certificate and key [either as a file path or as a PEM string](https://github.com/paullouisageneau/libdatachannel/blob/3a11fec497921fcb74c29c27d6e01b7071539c42/src/impl/websocketserver.cpp#L30).